<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModeDebugPluginWidget</class>
 <widget class="QWidget" name="ModeDebugPluginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="pushButtonUpdateVertexNormal">
     <property name="text">
      <string>update vertex normals</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonUpdateFaceNormal">
     <property name="text">
      <string>update face normals</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonUpdateBoundingBox">
     <property name="text">
      <string>update bounding box</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
